python/ray/rllib/es/es.py

```diff
@@ -170,7 +170,7 @@ def _init(self):
     def _collect_results(self, theta_id, min_episodes, min_timesteps):
         num_episodes, num_timesteps = 0, 0
         results = []
-        while num_episodes < min_episodes or num_timesteps < min_timesteps:
+        while num_episodes < min_episodes and num_timesteps < min_timesteps:
```

Comment:
This changes the meaning of the parameters. If the goal of this PR is to decrease the test durations, then we should do that by passing in something like

```
--config="{'episodes_per_batch': 10, 'timesteps_per_batch': 100}"
```

Reply:
Well, I'm not sure the parameters make sense in the first place. If you're training humanoid for example, the episodes get longer over time (something between 10-100x longer), so iterations get slower and slower, which is pretty weird.